libs/designsystem/slide/src/slides.component.html

```diff
@@ -1,5 +1,9 @@
 <div class="navigation" *ngIf="showNavigation || title !== undefined">
-  <span class="kirby-text-medium">{{ title }}</span>
+  <span class="kirby-text-medium">
+    <ng-content select="h1,h2,h3,h4,h5,h6">{{ title }}</ng-content>
+  </span>
+
+  <ng-content></ng-content>
```

Comment:
This line seems to have been accidentally added? 🤔 We already project the content via lines 55-57: 
```
<ng-container
      *ngTemplateOutlet="slideTemplate; context: { $implicit: slide, index: i }"
></ng-container>
```

```suggestion
```